monkey/infection_monkey/exploit/vsftpd.py

```diff
@@ -31,31 +31,31 @@ def __init__(self, host):
 
     def exploit_host(self):
         try:
-            LOG.info('[*] Attempting to trigger backdoor...')
+            LOG.info('Attempting to trigger backdoor...')
             ftp_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
             ftp_socket.connect((self.host.ip_addr, FTP_PORT))
-            ftp_socket.recv(1024).decode('utf-8')
+            ftp_socket.recv(128).decode('utf-8')
```

Comment:
Still don't understand why 128. Why this specific number or any other?

Reply:
Ah!  There is no specific length that needs to be there, as VakarisZ mentioned as a `case for race condition` , we just wait for the response from the server before sending the payload.

Reply:
Could have used 100, but I rounded it off to 128 ;)

Reply:
Export 128 as a global and explain that it's socket buffer size in bytes/bits.